coa-helm/templates/cromwell.yaml

```diff
@@ -0,0 +1,168 @@
+          command:
+            - /bin/sh
+            - -c
+          image: {{ .Values.images.cromwell }}
+          name: cromwell
+          stdin: true
+          tty: true
+          ports:
+            - containerPort: {{ .Values.service.cromwellPort }}
+          resources: {}
+          volumeMounts:
+            - mountPath: /cromwell-tmp
+              name: cromwell-tmp-claim
+            - mountPath: /configuration
+              name: configuration-claim1
+            - mountPath: /cromwell-executions
+              name: cromwell-executions-claim1
+            - mountPath: /cromwell-workflow-logs
+              name: cromwell-workflow-logs-claim1
+            - mountPath: /workflows
+              name: workflows-claim1
+            - mountPath: /inputs
+              name: inputs-claim1
+            - mountPath: /outputs
+              name: outputs-claim1
+            - mountPath: /configuration/cromwell
+              name: cromwell-config-volume
+      restartPolicy: Always
+      volumes:
+        - name: cromwell-tmp-claim
+          persistentVolumeClaim:
+            claimName: cromwell-tmp-claim
+        - name: configuration-claim1
+          persistentVolumeClaim:
+            claimName: configuration-claim1
+        - name: cromwell-executions-claim1
+          persistentVolumeClaim:
+            claimName: cromwell-executions-claim1
+        - name: cromwell-workflow-logs-claim1
+          persistentVolumeClaim:
+            claimName: cromwell-workflow-logs-claim1
+        - name: workflows-claim1
+          persistentVolumeClaim:
+            claimName: workflows-claim1
+        - name: inputs-claim1
+          persistentVolumeClaim:
+            claimName: inputs-claim1
+        - name: outputs-claim1
+          persistentVolumeClaim:
+            claimName: outputs-claim1
+        - name: cromwell-config-volume
+          configMap:
+            name: {{ include "app.fullname" . }}-cromwell-config
+status: {}
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "app.fullname" . }}-cromwell-svc
+  labels:
+  {{- include "app.labels" . | nindent 4 }}
+spec:
+  selector:
+  {{- include "app.cromwell.selectorLabels" . | nindent 4 }}
+  ports:
+    - name: {{ .Values.service.cromwellPort | quote }}
+      port: {{ .Values.service.cromwellPort }}
+      targetPort: {{ .Values.service.cromwellPort }}
+status:
+  loadBalancer: {}
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "app.fullname" . }}-cromwell-config
+data:
+  cromwell-application.conf: |-
+    include required(classpath("application"))
+
+    akka.http.host-connection-pool.pool-implementation = legacy
+
+    call-caching {
+      enabled = false
+    }
+
+    system {
+      input-read-limits {
+        lines = 1000000
+      }
+    }
+
+    engine {
+      filesystems {
+        local {
+          enabled: true
```

Comment:
I remember from one of the meetings with Microsoft there being a discussion of local filesystem access. Does this relate back to this? Should we enable local filesystem access?

Reply:
Right now this instance of Cromwell thinks it's running on a local filesystem, which is actually Blob containers mounted via [blobfuse](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-how-to-mount-container-linux). But soon... BlobFilesystem!